activities/recruitment/hiring-process.md

```diff
@@ -7,8 +7,8 @@ type: Resource
 ## Roles in this process
 
 * Head of human resources (HRM), responsible for the hiring process
+* Lead codebase steward, with a role in the first interview of applicants for codebase stewardship roles
```

Comment:
This can be the lead of any role, not just codebase steward, but for now this is OK.